src/pages/p2p/components/_dp2p.tsx

```diff
@@ -151,7 +151,7 @@ const DP2P = ({ P2P, reverse }: DP2PProps) => {
 
                 <VideoText>{localize('Find out how Deriv P2P works:')}</VideoText>
                 <StyledIFrame
-                    src="https://www.youtube.com/embed/zf9flqE94Ek"
+                    src="https://www.youtube.com/embed/sITTfAZAnwU"
```

Comment:
@adrienne-deriv so every time this changes we have to come and change it here too?

Reply:
Yes, currently its using an iframe linked to the youtube video (unless if we use CMS then we don't have to make this changes at here every time, only at the CMS dashboard)